### PR TITLE
Installed packages()

### PR DESCRIPTION
**Pull request by Baltmann01**

### CODE REVIEW

File: website/code_issues.qmd [hunk 1]
```diff
@@ -319,3 +319,44 @@ Calls of `rm(list = ls())` to remove variables of the current environment, shoul
...
+
+## Solution
+
+Instead of using `installed.packages()`, use `requireNamespace("pkg")` or `require("pkg")` to find out if packages are available.
+
+### Details
+
+::: {.callout-note title="CRAN Review Communication" appearance="simple" icon=false collapse=true}
+
+You are using `installed.packages()` in your code. As mentioned in the notes of `installed.packages()` help page, this can be very slow. Therefore do not use `installed.packages()`.
```
Comment (jasdumas) [inline website/code_issues.qmd:342]: Would it be helpful to link to this help page? https://stat.ethz.ch/R-manual/R-devel/library/utils/html/installed.packages.html

File: website/code_issues.qmd [hunk 2]
```diff
@@ -319,3 +319,44 @@ Calls of `rm(list = ls())` to remove variables of the current environment, shoul
...
+
+You are calling `installed.packages()` to check if a package is installed.
+
+## Solution
+
+Instead of using `installed.packages()`, use `requireNamespace("pkg")` or `require("pkg")` to find out if packages are available.
+
+### Details
+
+::: {.callout-note title="CRAN Review Communication" appearance="simple" icon=false collapse=true}
```
Comment (jasdumas) [inline website/code_issues.qmd:340]: Screenshot 2024-10-02 at 10 39 14 AM

I'm also not seeing the callout format rendered correctly.

**Review by jasdumas — COMMENTED** [review]
I left a comment around formatting and adding a link 😄

File: website/code_issues.qmd (hunk 1 above)
Comment (Baltmann01) [reply website/code_issues.qmd:342]: I added it in the first paragraph of the "Details"

File: website/code_issues.qmd (hunk 2 above)
Comment (Baltmann01) [reply website/code_issues.qmd:340]: Weird, I'll look into this and fix it

Comment (Baltmann01) [reply website/code_issues.qmd:340]: It seems to work in the rendered webpage. No idea what went wrong in your render

**Pull request merged**